src/pages/HttpBin/index.js

```diff
@@ -45,28 +45,27 @@ const HttpBin = () => {
   const [responseIsLoad, setResponseIsLoad] = useState(true);
 
   const createRequest = () => {
```

Comment:
you have to execute a function that sets isLoading to true

```js
setResponseIsLoad(true);
```

Am I Right?

Reply:
You are right. update will be in the next commit